tools/roslaunch/src/roslaunch/cached.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import roslib.packages
+import catkin.find_in_workspaces
+from catkin.find_in_workspaces import collect_path_data_in_workspaces
```

Comment:
As mentioned in the catkin PR this function should be implemented here rather then in catkin itself. Also only a part of the return values is used so the function can be simplified to only return these information.